src/client/components/threatmodal/threatmodal.jsx

```diff
@@ -76,13 +81,145 @@ class ThreatModal extends React.Component {
+            name="type"
+            id="type"
+            disabled={!this.isOwner}
+            value={this.props.G.threat.type}
+            onChange={(e) =>
+              this.props.moves.updateThreat('type', e.target.value)
+            }
+          >
+            {getSuits(this.props.G.gameMode).map((suit) => (
+              <option value={suit} key={`threat-category-${suit}`}>
+                {getSuitDisplayName(this.props.G.gameMode, suit)}
+              </option>
+            ))}
+          </Input>
+        </FormGroup>
+        <FormGroup>
+          <Label for="severity">Severity</Label>
+          <Input
+            type="select"
+            name="severity"
+            id="severity"
+            disabled={!this.isOwner}
+            value={this.props.G.threat.severity}
+            onChange={(e) =>
+              this.props.moves.updateThreat('severity', e.target.value)
+            }
+          >
+            <option>Low</option>
+            <option>Medium</option>
+            <option>High</option>
+          </Input>
+        </FormGroup>
+        <FormGroup>
+          <Label for="description">Description</Label>
+          <Input
+            type="textarea"
+            name="description"
+            id="description"
+            disabled={!this.isOwner}
+            style={{ height: 150 }}
+            value={this.state.description}
+            onBlur={(e) =>
+              this.props.moves.updateThreat('description', e.target.value)
+            }
+            onChange={(e) => this.setState({ description: e.target.value })}
+          />
+        </FormGroup>
+        <FormGroup hidden={!this.isOwner}>
+          <div className="checkbox-item">
+            <Input
+              className="pointer"
+              type="checkbox"
+              id="showMitigation"
+              onChange={(e) => this.toggleMitigationField(e.target.checked)}
+            />
+            <Label for="showMitigation">
+              Add a mitigation <em>(optional)</em>
+            </Label>
+          </div>
+        </FormGroup>
+        <FormGroup hidden={this.isOwner && !this.state.showMitigation}>
+          <Label for="mitigation">Mitigation</Label>
+          <Input
+            type="textarea"
+            name="mitigation"
+            id="mitigation"
+            disabled={!this.isOwner}
+            style={{ height: 150 }}
+            value={this.state.mitigation}
+            onBlur={(e) =>
+              this.props.moves.updateThreat('mitigation', e.target.value)
+            }
+            onChange={(e) => this.setState({ mitigation: e.target.value })}
+          />
+        </FormGroup>
+      </ModalBody>
+    );
+  }
+
+  threatRestrictedDetailModalBody() {
+    return (
+      <ModalBody>
+        <FormGroup>
+          <Label for="ref">
+            Reference <em>(e.g. link to external bug tracking system)</em>
+          </Label>
+          <Input
+            type="text"
+            name="ref"
```

Comment:
Similar to my previous comment, I would choose another, more descriptive name. Even just `reference` would be better and less confusing the the context of React.

Reply:
done